mock/cpp_mock_scenarios/src/metrics/momentary_stop.cpp

```diff
@@ -29,10 +28,7 @@ class MomentaryStopScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit MomentaryStopScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "momentary_stop",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.